macOS/LocalPackages/NewTabPage/Sources/NewTabPage/Omnibar/NewTabPageDataModel+Omnibar.swift

```diff
@@ -0,0 +1,196 @@
+//
+//  NewTabPageDataModel+Omnibar.swift
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+public extension NewTabPageDataModel {
+
+    // https://github.com/duckduckgo/content-scope-scripts/blob/cb36865bf4087456dd3da282bf49ccb59cbfcc47/special-pages/pages/new-tab/app/omnibar/omnibar.md
```

Comment:
Good idea to link documentation in these files!

Here’s the HTML docs link that will always be up-to-date: https://duckduckgo.github.io/content-scope-scripts/documents/New_Tab_Page.Omnibar_Widget.html